### PR TITLE
Miniconda to Micromamba installation guide

### PR DESCRIPTION
**Pull request by uttiyasarkar**
- Faster installation of packages while using micromamba over miniconda
- Updated README.md for using micromamba installation, miniconda installation removed from instructions

### CODE REVIEW

**Review by Ming-Yan — APPROVED** [review]
Hi @uttiyasarkar thanks for the changes, the PR looks good to me. merged to the central repo!

File: README.md [hunk 1]
```diff
@@ -23,19 +23,19 @@ git clone --recursive git@github.com:cms-btv-pog/BTVNanoCommissioning.git
 # activate enviroment once you have coffea framework 
 conda activate btv_coffea
 ```
-### Coffea installation with Miniconda
-For installing Miniconda, see also https://hackmd.io/GkiNxag0TUmHnnCiqdND1Q#Local-or-remote
+### Coffea installation with Micromamba
+For installing Micromamba, see [[here](https://mamba.readthedocs.io/en/latest/installation/micromamba-installation.html)]
 ```
-wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+wget wget -L micro.mamba.pm/install.sh
```
Comment (Ming-Yan) [inline README.md:29]: is the `wget` duplicated?

**Review by Ming-Yan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (uttiyasarkar) [reply README.md:29]: Accidentally duplicated. It is a typo

Comment (Ming-Yan) [reply README.md]: Thank you for changing this! will merge :)

**Review by Ming-Yan — APPROVED** [review]
Thanks @uttiyasarkar taking care of this! will merge to the central repo!

**Pull request merged**